docs/reference/cmd/pipeline_start.md

```diff
@@ -14,11 +14,16 @@ To run a Pipeline that has one git resource and no parameter.
 	$ tkn pipeline start --resource source=samples-git
 
 
-To run a Pipeline that has one git resource, one image resource and
-two parameters (foo and bar)
+To run a Pipeline that has one git resource, one image resource,
+two parameters (foo and bar) and four workspaces (my-config, my-pvc,
+my-secret and my-empty-dir)
 
 
 	$ tkn pipeline start --resource source=samples-git \
 		--resource image=my-image \
 		--param foo=yay \
-		--param bar=10
+		--param bar=10 \
+		--workspaces name=my-secret,secret=password \
```

Comment:
how about we call it --workspace (singular) instead of `workspace*s*` ?

Reply:
I agree with Sunil... let's keep the parameter naming consistent.